src/graph/client/graphClient.ts

```diff
@@ -131,14 +156,24 @@ export class GraphClient {
 
   public selectGraphView() {
     this._graphView = true;
-    d3.select(htmlElements.graphSection).classed("active", true);
-    d3.select(htmlElements.jsonSection).classed("active", false);
+    this.setView();
   }
 
   public selectJsonView() {
     this._graphView = false;
-    d3.select(htmlElements.graphSection).classed("active", false);
-    d3.select(htmlElements.jsonSection).classed("active", true);
+    this.setView();
+  }
+
+  public setQuery(query: string) {
+    this.emitToHost('setQuery', query);
+  }
+
+  private setView() {
+    htmlElements.graphRadio.checked = this._graphView;
```

Comment:
You should call selectGraphView or selectJsonView() instead.

Reply:
Both select view functions call this.